### PR TITLE
Quick portal redesign

### PR DESCRIPTION
**Pull request by megalodonatron**
Updates the way that portal currently looks without changing any underlying mechanisms.

Changelog:

Portal got a makeover :rabbit2:

Updated language
-add sample is now log sample
-participants are now human samples

Updated menu organization

### CODE REVIEW

**Comment by wasade** [comment]
Any success on getting the step-by-step info on the landing page up?

**Comment by megalodonatron** [comment]
That's what this is.

**Comment by wasade** [comment]
Sorry, thanks -- commit comments were not to that effect so I was confused. Will pull down today and check. 

@douginator2000 are you able to pull down and look at this as well?

**Comment by douginator2000** [comment]
If someone can just pull it to webdev we can all look at it.

On Fri, Jul 5, 2013 at 12:54 PM, Daniel McDonald
notifications@github.comwrote:

> Sorry, thanks -- commit comments were not to that effect so I was
> confused. Will pull down today and check.
> 
> @douginator2000 https://github.com/douginator2000 are you able to pull
> down and look at this as well?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/qiime/qiime_web_app/pull/459#issuecomment-20533858
> .

## 

Doug Wendel
Professional Research Associate
University of Colorado at Boulder
Department of Chemistry and Biochemistry, Knight Lab
303-641-5442
wendel@colorado.edu

**Comment by wasade** [comment]
We'll have to wait until QB is back up, hiccup on the dev database

**Comment by wasade** [comment]
This is moving in the right direction, thanks Meg! Below are some comments
-Daniel

Login page is off (see image)

Verification isn't centered (see image). Also, the unverified kit has a
green checkmark in the tab box indicating it is complete (possibly a
caching affect?)

Adding an animal or environmental as a sample source takes you back to step
1

Steps 2 and 3 are confusing in that, if you add yourself as a human in step
2, then step 3 is redundant. Also, please put a large and red EXAMPLE, or
something else to visually indicate that you should not be able to click on
fields in it

Step 4, please add images of bad fecal samples and the images for
collecting oral/forehead. Also " screw the red handling cap on firmly", the
sample tubes aren't threaded AFAIK

Step 4/5, add mention that if the sample cannot be shipped to store it in a
cool dry place such as a cabinet, and that they do not refrigerate or
freeze the sample

Step 6, where "8 to 10 months to finish processing." come from?

**Comment by adamrp** [comment]
This looks _tons_ better, thanks, Meg!

@wasade, I think the green check mark does not mean that the step is complete, it's just the graphic to communicate "verification step" -- this is kinda confusing and should probably be changed.

I think the animal survey didn't exist yet when this redesign was made.  The link should point to animal_survey.psp (there is still a pull request in for the animal survey, too, which needs to be reviewed and merged)

I would merge steps 2 and 3 into a single tab.  Put the text from step 3 into step 2's tab, and remove the screenshot in favor of a list (full or partial) of the questions.

I think we should change the "Hi, Test!" Welcome message to include something along the lines of "Please follow the step-by-step instructions below"

I think we should move the list of participants and samples off the side menu and into the main portal somehow.  One idea would be to change the title of the step 2 from "Add Sample Source" to just "Sample Sources," and then the 3 columns could list the sample sources of each type (human, animal, environmental).

Which tab is displayed upon login should be dependent on kit verification status: if the kit is not verified, load tab 1 automatically; if the kit is already verified, load tab 2 automatically.  In fact, we might just make step 1 grayed out if the kit is verified already.

File: www/americangut/source_portal.psp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<h2>Sample Source</h2>
+<p class="portal_blurb">
+There are three different sample sources that you can choose from for the American Gut Project. The sources are human, animal and environmental. The buttons below will allow you to add a new sample source.</p>
+<p class="portal_blurb">
+If you add a <strong>human</strong> source, you will need to have a completed 7 day food diary for that human. If you add a <strong>human</strong> or <strong>animal</strong> source, you will be asked to complete a survey. You can read more about the survey on <a href="javascript:selectTab('survey')">the survey tab</a>.
+</p>
+<div id="source_icons">
+    <a href="fusebox.psp?page=new_participant_overview.psp"><div class="source_icon"><img src="img/human.png"><div>Human</div></div></a>
+    <a href=""><div class="source_icon"><img src="img/animal.png"><div>Animal</div></div></a>
```
Comment (adamrp) [inline www/americangut/source_portal.psp]: This should be a href="fusebox.psp?page=animal_survey.psp" (note that the animal_survey.psp page is in a pending pull request #449)

File: www/americangut/portal.psp [hunk 2]
```diff
@@ -20,45 +20,36 @@ if message:
...
+            <li id="source_tab" class="unselected" style="background-image: url('img/2.png');"><a href="javascript:selectTab('source')">Add a<br>Sample Source<br><img src="img/source.png"/></a></li>
+            <li id="survey_tab" class="unselected" style="background-image: url('img/3.png');"><a href="javascript:selectTab('survey')">Complete Survey<br><br><img src="img/survey.png"/></a></li>
+            <li id="sample_tab" class="unselected" style="background-image: url('img/4.png');"><a href="javascript:selectTab('sample')">Take &amp; Log a<br>Sample<br><img src="img/sample.png"/></a></li>
+            <li id="mail_tab" class="unselected" style="background-image: url('img/5.png');"><a href="javascript:selectTab('mail')">Mail Samples<br>to Us<br><img src="img/mail.png"/></a></li>
+            <li id="results_tab" class="unselected" style="background-image: url('img/6.png');"><a href="javascript:selectTab('results')">Sequencing &amp;<br>Results<br><img src="img/results.png"/></a></li>
+        </ul>
+        <div id="workflow_content">
+            <div id="verify_content" class="portal_selected">
```
Comment (adamrp) [inline www/americangut/portal.psp:59]: make class here dependent on whether or not the kit has already been verified

**Comment by megalodonatron** [comment]
> Daniel:
> Step 4, please add images of bad fecal samples and the images for
> collecting oral/forehead.

I don't have good resolution files of those images, if someone can get them to me I will add them

> Daniel:
> Step 6, where "8 to 10 months to finish processing." come from?

That refers specifically to oral/skin samples and that quote came from Greg

**Comment by rob-knight** [comment]
That had really better not be true -- what is the basis for it? Does it still hold with the transition to MiSeq?

On Jul 15, 2013, at 4:18 PM, Meg Pirrung <notifications@github.com<mailto:notifications@github.com>> wrote:

Daniel:
Step 4, please add images of bad fecal samples and the images for
collecting oral/forehead.

I don't have good resolution files of those images, if someone can get them to me I will add them

Daniel:
Step 6, where "8 to 10 months to finish processing." come from?

That refers specifically to oral/skin samples and that quote came from Greg

**Comment by wasade** [comment]
Hires good/bad fecal samples. I pulled the oral/forehead images for the instructions from previous materials you had put together
bad1
good1
bad2
good2

**Comment by megalodonatron** [comment]
I added a button to the mail page but it's commented out right now. If someone could add a field to the db and make a stored procedure to set that variable that would be awesome.

**Comment by RNAer** [comment]
+1. It looks much better.

would the text be better to be left aligned intead of center aligned? or
it's just me?

**Comment by adamrp** [comment]
Looks very nice, and continues to improve!  Just a couple comments:

1) If the kit is verified, I think the first tab should be disabled.  Or, if not disabled, there should be a message that shows up when that tab is selected that says "Your kit has been verified!"
2) In the last tab (sequencing and results), are the time estimates accurate?  I seem to remember Rob and Daniel having some concerns there

**Comment by megalodonatron** [comment]
> Zech
> would the text be better to be left aligned intead of center aligned?

Can you take a screenshot? it should all be left aligned

> Adam
> 1.

I will work on making it disabled/grayed out

> 2.

I have not yet received an updated time frame, maybe that part should just be removed for now?

**Comment by mikewise** [comment]
I believe the 8-10 month estimate was due to a low expected volume of
samples returned by participants. One thing that we can do in the plate map
side is combine oral and skin in one plate separated by a row of blanks to
accomodate a lower influx of those sample types. I think that we can do
oral and skin samples in the same time period as we can do fecal samples.

**Comment by adamrp** [comment]
Cool!  Last thing from me: can you make step 1 not highlight on mouseover if it is disabled?

**Comment by RNAer** [comment]
@meganap 
screen shot 2013-07-16 at 12 45 09 pm

**Comment by megalodonatron** [comment]
> Adam
> Last thing from me: can you make step 1 not highlight on mouseover if it is disabled?

I tried, it's not possible with the way that the css currently works, since I'm just applying the same styles to each different tab. The only other option is to not have anything light up :/

> Zech's screenshot

Looks like the css is not loading properly for you, what browser are you using?

**Comment by RNAer** [comment]
firefox. right, the css somehow did not reload. Everything looks great
after a hard refresh.

**Comment by douginator2000** [comment]
screen shot 2013-07-17 at 12 34 03 pm

Noticing that the right margin appears to be 0, making the text run directly into the div border.

**Comment by douginator2000** [comment]
On webdev now.

**Pull request merged**